pkg/tenantctl/controller.go

```diff
@@ -635,23 +635,23 @@ func (c *Controller) deleteRoleBinding(roleBinding *rbacv1beta1.RoleBinding) err
 	return nil
 }
 
-func (c *Controller) logPrintln(tenant *api.Tenant, v ...interface{}) {
-	log.Printf(
+func (c *Controller) formatLog(tenant *api.Tenant, v ...interface{}) string {
+	return fmt.Sprintf(
 		"client %q: tenant %q: %s",
 		tenant.GetNamespace(), tenant.GetName(),
 		fmt.Sprint(v...),
 	)
 }
 
-func (c *Controller) logPrintf(tenant *api.Tenant, format string, v ...interface{}) {
-	c.logPrintln(tenant, fmt.Sprintf(format, v...))
+func (c *Controller) formatLogf(tenant *api.Tenant, format string, v ...interface{}) string {
+	return c.formatLog(tenant, fmt.Sprintf(format, v...))
 }
 
 func (c *Controller) updateMetrics() {
 	// TODO determine number of tenants per client
 	list, err := c.tenantLister.List(labels.Everything())
 	if err != nil {
-		log.Printf("Cannot update tenant metrics: %s", err.Error())
+		klog.V(2).Infof("Cannot update tenant metrics: %s", err.Error()) //TODO log Error
```

Comment:
Should be logged as ERROR, not INFO.

Reply:
Like this?
```suggestion
		klog.V(1).ErrorS(err, "Cannot update tenant metrics")
```
https://github.com/kubernetes/klog/blob/master/klog.go#L1353

Reply:
Yes, but unconditionally:
```suggestion
		klog.ErrorS(err, "Cannot update tenant metrics")
```

Reply:
Why still "Infof"?